node/status_node_services.go

```diff
@@ -473,10 +473,17 @@ func (b *StatusNode) appmetricsService() common.StatusService {
 
 func (b *StatusNode) walletService(accountsDB *accounts.Database, accountsFeed *event.Feed, openseaAPIKey string) common.StatusService {
 	if b.walletSrvc == nil {
+		var extService *ext.Service
+		if b.WakuV2ExtService() != nil {
```

Comment:
You check WakuV2ExtService twice

Reply:
🤦

Reply:
We don't use WakuV1 anymore, so I expect the latter check not to help.